Gemfile

```diff
@@ -1,24 +1,21 @@
+# frozen_string_literal: true
 source 'https://rubygems.org'
-
 ruby '3.2.2'
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
 gem 'rails', '~> 7.1.2'
 
 # The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
 gem 'sprockets-rails'
-
 # Use postgresql as the database for Active Record
 gem 'pg', '~> 1.1'
-
 # Use the Puma web server [https://github.com/puma/puma]
 gem 'puma', '>= 5.0'
-
 # Use JavaScript with ESM import maps [https://github.com/rails/importmap-rails]
 gem 'importmap-rails'
-
-# Hotwire's SPA-like page accelerator [https://turbo.hotwired.dev]
+# Hotwire's SPA-lke page accelerator [https://turbo.hotwired.dev]
 gem 'turbo-rails'
+gem 'tzinfo-data'
```

Comment:
What is the purpose of this?

Reply:
Bhaiya, I did not include it directly. I automatically added it when I reinstalled my project.